test/utils/test_cost_funcs.py

```diff
@@ -44,6 +45,10 @@ def test_cost_func_rzx(self):
         qc.rzx(theta, 0, 1)
         qc.rzx(phi, 1, 2)
 
+        qc2 = deepcopy(qc)
```

Comment:
Is `qc.copy()` sufficient? It's better to rely on copy methods provided by the classes themselves rather than a general one like `deepcopy`.

Reply:
Correct, fixed.